pyproject.toml

```diff
@@ -29,13 +29,12 @@ viz-plotly = ["networkx>=2", "plotly>=4", "pydot"]
 all = ["networkx>=2", "plotly>=4", "pydot", "tiledb-plot-widget>=0.1.7"]
 
 dev = ["black", "pytest", "ruff"]
-tests = ["xarray", "pytest-split"]
+tests = ["xarray", "pytest-cov", "pytest-explicit", "pytest-split"]
```

Comment:
We have some config that depends on pytest-explicit and are using pytest-cov in CI.